src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/IntegrationTestBase.kt

```diff
@@ -932,3 +948,17 @@ abstract class IntegrationTestBase {
   fun loadPreemptiveCacheForOffenderDetails(crn: String) = communityApiClient.getOffenderDetailSummaryWithCall(crn)
   fun loadPreemptiveCacheForInmateDetails(nomsNumber: String) = prisonsApiClient.getInmateDetailsWithCall(nomsNumber)
 }
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@Tag("isPerClass")
+abstract class PerClassIntegrationTestBase : IntegrationTestBase() {
```

Comment:
it may be more descriptive to call this something like 'InitialiseDatabasePerClassTestBase'?